### PR TITLE
[CIFuzz] Add seed and len control to fuzzer arguments

### PR DESCRIPTION
**Pull request by Leo-Neat**
Adds seed number and length control arguments when calling running a fuzz target. These two arguments are added to optimize for time constrained fuzzing.  Detailed info about the arguments can be found here: https://llvm.org/docs/LibFuzzer.html#options

### CODE REVIEW

File: infra/cifuzz/fuzz_target.py [hunk 1]
```diff
@@ -72,7 +73,9 @@ def fuzz(self):
     command += [
         '-e', 'FUZZING_ENGINE=libfuzzer', '-e', 'SANITIZER=address', '-e',
         'RUN_FUZZER_MODE=interactive', 'gcr.io/oss-fuzz-base/base-runner',
-        'bash', '-c', 'run_fuzzer {0}'.format(self.target_name)
+        'bash', '-c', 'run_fuzzer {0} {1} {2}'.format(self.target_name,
```
Comment (Dor1s) [inline infra/cifuzz/fuzz_target.py]: nit: let's use named arguments when there are many of them, e.g. `'{fuzz_target} {options}'.format(fuzz_target=self.target_name, options='-seed=1337 -len_control=0'`

Comment (Leo-Neat) [reply infra/cifuzz/fuzz_target.py]: Fixed

File: infra/cifuzz/fuzz_target.py [hunk 2]
```diff
@@ -72,7 +73,9 @@ def fuzz(self):
     command += [
         '-e', 'FUZZING_ENGINE=libfuzzer', '-e', 'SANITIZER=address', '-e',
         'RUN_FUZZER_MODE=interactive', 'gcr.io/oss-fuzz-base/base-runner',
-        'bash', '-c', 'run_fuzzer {0}'.format(self.target_name)
+        'bash', '-c', 'run_fuzzer {fuzz target} {seed} {len control}'.format(self.target_name,
```
Comment (Dor1s) [inline infra/cifuzz/fuzz_target.py]: I don't think it works this way, the arguments passed to the `format()` function all should be named (i.e. `.format(fuzz_target=self.target_name, ...)`

Also, I wasn't explicit on that in my previous comment, let's put both options into a single `options` parameter.

The constant string `-seed=1337 -len_control=0` can go into some global constant (e.g. `DEFAULT_OPTIONS = '-seed=1337 -len_control=0'`) and we may want to make it possible to modify in future

Comment (Leo-Neat) [reply infra/cifuzz/fuzz_target.py]: That makes a lot of sense, thanks.

File: infra/cifuzz/fuzz_target.py [hunk 3]
```diff
@@ -27,6 +28,8 @@
     format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
     level=logging.DEBUG)
 
+DEFAULT_OPTIONS = '-seed=1337 -len_control=0'
```
Comment (jonathanmetzman) [inline infra/cifuzz/fuzz_target.py]: nit: name this LIBFUZZER_OPTIONS.

**Pull request merged**